src/pages/bookmark/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect } from "react";
+import Spacer from "@components/base/Spacer";
+import { type } from "os";
+import Link from "next/link";
+import styled from "@emotion/styled";
+import { NextPage } from "next";
+import ShareButton from "@components/ShareButton";
+
+interface BasketballCourt {
+  id: number;
+  name: string;
+  address: string;
+}
+declare global {
+  interface Window {
+    Kakao: any;
+  }
+}
+
+type BasketballCourts = BasketballCourt[];
+
+const Bookmark: NextPage = () => {
+  useEffect(() => {
+    if (typeof window !== "undefined") {
+      window.Kakao.init("c6f32516ffb011a356a9f8ea036ca21f"); // TODO env 파일로 바꾸기
+    }
+  }, []);
+  const basketballCourts = [
+    {
+      id: 1,
+      name: "용왕산 근린 공원 농구장",
+      address: "서울 양천구 신곡 2동 3002번지 19번지 19번지 19번지 199-51",
+    },
+    {
+      id: 2,
+      name: "백두산 근린 공원 농구장",
+      address: "서울 양천구 신곡 2동 3002번지 19번지 19번지 19번지 199-51",
+    },
+    {
+      id: 3,
+      name: "북한산 근린 공원 농구장",
+      address: "서울 양천구 신곡 2동 3002번지 19번지 19번지 19번지 199-51",
+    },
+  ];
+
+  if (basketballCourts.length === 0) {
+    return (
+      <>
+        <Spacer>
+          <div>즐겨찾는 농구장이 없으시네요?</div>
+          <Link href="/map">
+            <button>나의 농구장 찾기</button>
+          </Link>
+        </Spacer>
+      </>
+    );
+  }
+
+  return (
+    <Spacer>
+      {basketballCourts.map((basketballCourt) => (
+        <BorderDiv key={basketballCourt.id}>
+          <p>{basketballCourt.name}</p>
+          <p>{basketballCourt.address}</p>
+          <button>즐겨찾기</button>
+          <Link href="/chat">
+            <button>채팅</button>
+          </Link>
+          <ShareButton />
+          <Link href="/reserve">
+            <button>예약하기</button>
+          </Link>
+        </BorderDiv>
+      ))}
```

Comment:
요 경우에는 이렇게 쓰면 훨씬 더 편할 것 같아요!
```suggestion
      {basketballCourts.map(({id, name, address}) => (
        <BorderDiv key={id}>
          <p>{name}</p>
          <p>{address}</p>
          <button>즐겨찾기</button>
          <Link href="/chat">
            <button>채팅</button>
          </Link>
          <ShareButton />
          <Link href="/reserve">
            <button>예약하기</button>
          </Link>
        </BorderDiv>
      ))}
```